odata_query/sqlalchemy/__init__.py

```diff
@@ -1,2 +1,10 @@
-from .shorthand import apply_odata_query
+from .shorthand import apply_odata_core, apply_odata_query
 from .sqlalchemy_clause import AstToSqlAlchemyClauseVisitor
+from .sqlalchemy_core import AstToSqlAlchemyCoreVisitor
+
```

Comment:
Is there any reason to include the `__all__`? 
As far as I understand it's redundant in this case, since the imports are the only names in this `__init__` module. Feel free to correct me if I'm wrong!

Reply:
Yes!  My apologies but my linter was complaining about these unused imports, and given the actual reason for importing them is to export them, adding them via ``__all__`` silenced these linter errors.